src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
```

Comment:
- [ ] It is part of project requirement to have a reduce for the category too as you have it for the books. You're expect to do the same thing you did for the books for the category. Kindly check the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) for more clarification.

![project-rqtm](https://user-images.githubusercontent.com/26597432/207302083-1ff4f049-05c3-4325-9f74-ead01fbf4300.png)